libfreerdp/core/capabilities.c

```diff
@@ -113,7 +113,6 @@ static BOOL rdp_read_capability_set_header(wStream* s, UINT16* length, UINT16* t
 
```

Comment:
Not all functions where touched, but ensure that all get and set methods are using WINPR_ASSERT before accessing a pointer, check the whole file!